datascience_eda/datascience_eda.py

```diff
@@ -165,3 +170,33 @@ def eda_text_columns(df, text_col = None, params = dict()):
     # plot a bar chart of most common tokens per entity
 
     # plot a bar chart of Part-of-speech tags
+
+
+def eda_categorical_columns(df, categorical_cols):
+    """Perfroms EDA of categorical features.
+    - Creates a dataframe containing column its details about unique values, null values and most frequenct category in the column
+    - Plots count plots for given categorical columns
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        the dataset (X)
+    categorical_col : list
+        name of categorical column(s)
+        
+    Returns
+    -------
+    dataframe
```

Comment:
any example of how to use the result? should we just return a list of plots generated for simplicity?